instance/models/openedx_appserver.py

```diff
@@ -340,22 +340,26 @@ def smtp_relay_settings(self):
     @property
     def admin_users(self):
         """
-        Return GitHub usernames, if exists, of admin users for this instance.
+        Retrieve all users that fulfill requirements defined via `admin_user_query`,
+        excluding any users that don't have a meaningful GitHub username.
+        (Without a meaningful GitHub username it won't be possible
+        to correctly set up SSH access to a given sandbox.
```

Comment:
@ahmedaljazzar Shoot, this last sentence is missing a closing paren. Looks like I forgot to include that [here](https://github.com/open-craft/opencraft/pull/354#discussion_r225066048).